### ISSUE TITLE
search kippt when searching google

### ISSUE THREAD

**Issue by kylewest** [kippt/kippt-chrome#5]
This is a cool feature in the evernote extension. A standard omnibox search simultaneously searches an evernote account with results displayed inline.

![](http://cl.ly/image/2I1w2F45360k/search-evernote.png)

**Is anyone working on this or something similar?** If not I'll try to implement and send a pull request.

Kyle

**Comment by ksaa** [comment]
At least I haven't heard anyone mention about it.

**Comment by jorilallo** [comment]
Yep, haven't heard but I think it would be useful and also pretty easy to implement.

**Comment by kylewest** [comment]
@jorde I'll have some free time over the holidays, could you point me in the right direction to _easy_?

**Comment by jorilallo** [comment]
@kylewest This is pretty straightforward: You need to inject code to the right place on google.com (content scripts). I would do another page and load that as an iframe inside the extension which then loads search results via ajax call (https://kippt.com/api/search/clips/?q={{search_term}}).

I almost think this should be something I should do as it could be integral part of the extension but feel free to hack around with it, I'm too busy at the moment :(

**Comment by kylewest** [comment]
thanks @jorde 

I haven't worked on any extensions so I didn't realize you could just call the api via ajax. I figured there would be a bunch of cross-domain issues. With that in mind it does become very straightforward. I'll hack on it and maybe you can use it, or parts of it, when you come back around to this.

The evernote implementation of this concept is nice, although their search leaves much to be desired.

![Screenshot 12:14:12 9:05 AM](https://f.cloud.github.com/assets/6247/13324/cce9cb7a-45f7-11e2-9021-d86d579f6fbe.png)

Kyle

**Comment by jorilallo** [comment]
Yeah, Chrome extensions are pretty nice as there's no cross-domain issues if you just define the urls in config file.

We're going to change the look&feel of Kippt quite a lot but I think simple list of results would be the best way to represent results. Probably the biggest issue with this is to inject iframe into right place so that it won't affect Google's functionality otherwise.

We appreciate all the help and would love to see this happening soon :+1:

**Comment by whispy** [comment]
@kylewest  @jorilallo  any news on this feature?
